docs/general/builtins/scalar-functions.rst

```diff
@@ -958,6 +981,10 @@ The following fields are supported:
   Year 2000 century 20 and year 2001 is also century 20. This is different to
   the GregorianJulian (GJ) calendar system where 2001 would be century 21.
 
+  .. NOTE::
```

Comment:
here and below, maybe a note on each is too much, I could also use backquotes and have the comment just in italics.

Reply:
I think the unsupported fields are self explanatory that it won't work with intervals.

Reply:
Yeah, maybe it's too much, I can only leave the comment on `DAY` which has different semantics for timestamps vs intervals. @mfussenegger do you agree?